resources/entitlements.mas.plist

```diff
@@ -8,5 +8,10 @@
 	<true/>
 	<key>com.apple.security.files.user-selected.read-write</key>
 	<true/>
+	<key>com.apple.security.temporary-exception.files.home-relative-path.read-write</key>
+	<array>
+		<string>/Library/Application Support/Mozilla/NativeMessagingHosts/</string>
+		<string>/Library/Application Support/Google/Chrome/NativeMessagingHosts/</string>
```

Comment:
@Hinton Do we need paths for other browsers here too? like edge chromium and opera?

Reply:
They should use the same as chrome in my experience. Not 100% sure on Edge though.

Reply:
Seems like edge on mac uses a different directory, I'll make a follow up PR for this.